File: javascript/Day_1/Question_17.js
number = 5;

const Fibonnaci = () => {
  a = 0;
  b = 1;
  for (let i = 1; i < number + 1; i++) {
    temp = a + b;
    a = b;
    b = temp;
    console.log(`${b}`);
  }
};

Fibonnaci();

File: dsa/Day_1/Question_17.cpp
#include<iostream>
using namespace std;

void Fibonnaci(int n){
    int a = 0;
    int b = 1;
    for (int i = 0; i <= n; i++)
    {
        int temp = a + b;
        a = b;
        b = temp;

        cout << "Fibonnaci: " << b << endl;
    }
}

int main(){
    int n;
    cout << "n: ";
    cin >> n;
    Fibonnaci(n);

    return 0;
}